packages/analytics/dashboard-renderer/sandbox/App.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="sandbox-container">
```

Comment:
Could you use the new sandbox pattern which uses the public <SandboxLayout> shell component with slots for content and controls section? 
  
 See how the sandbox is setup in `analytics-chart`